### ISSUE TITLE
Gyroflow OFX plugin loads, but has no effect on video

### ISSUE THREAD

**Issue by jimbodude** [gyroflow/gyroflow-ofx#47]
Gyroflow OFX 1.2.2
Gyroflow 1.5.2
Davinci 18.6 build 9 (free, not studio)
macOS Sonoma 14.0
MacBook Pro, 2021 M1 Pro

I have `/Library/OFX/Plugins/Gyroflow.ofx.bundle`
```shell
ls -alh /Library/OFX/Plugins/Gyroflow.ofx.bundle
total 0
drwxr-xr-x@ 3 j     wheel    96B Jul 13 13:11 .
drwxr-xr-x  3 root  wheel    96B Sep 24 15:10 ..
drwxr-xr-x@ 7 j     wheel   224B Jul 13 13:11 Contents
```

Davinci loads the plugin, and I'm able to apply it in Edit and create a node with it in Fusion or Color.

In Edit or Color: I can load the gyroflow project and the checkbox "OK" comes up.  I can move the sliders and see the check box change from "calculating..." to "OK".  However, the preview in the node remains a grey box, and no updates occur on the main preview window.

In Fusion: The gyroflow node and MediaOut1 node flash from red to yellow very quickly until the project is loaded, which must be done using the "browse" button, not pasting the file name.  Once loaded, the behavior is the same as in Edit and Color - settings have no effect.

On the Deliver tab, I'm able to render the video, but no gyroflow settings are applied to the output.

Under System Settings / Privacy & Security / Developer Tools, I've enabled DaVinci Resolve under "Allow the applications to run software locally that does not meet the system's security policy" - this setting had no effect.  On System Settings / Privacy & Security I have "Allow applications downloaded from" set to "App store and identified developers".  There does not appear to be a blanket "allow unknown developers" feature in Sonoma.

The log is filled with errors like:
```
0x30d39f000    | Fusion               | ERROR | 2023-09-29 05:28:53,944 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionGetFramesNeeded
0x30d0bf000    | Fusion               | ERROR | 2023-09-29 05:28:53,945 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionRender
0x30d39f000    | Fusion               | ERROR | 2023-09-29 05:28:53,947 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionGetFramesNeeded
0x2aeeab000    | Fusion               | ERROR | 2023-09-29 05:28:53,948 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionRender
0x30d39f000    | Fusion               | ERROR | 2023-09-29 05:28:53,951 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionGetFramesNeeded
0x30d0bf000    | Fusion               | ERROR | 2023-09-29 05:28:53,951 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionRender
0x30d0bf000    | Fusion               | ERROR | 2023-09-29 05:28:53,954 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionGetFramesNeeded
0x30d0bf000    | Fusion               | ERROR | 2023-09-29 05:28:53,955 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionRender
0x30d39f000    | Fusion               | ERROR | 2023-09-29 05:28:53,956 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionGetFramesNeeded
0x2aeeab000    | Fusion               | ERROR | 2023-09-29 05:28:53,957 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionRender
0x30d39f000    | Fusion               | ERROR | 2023-09-29 05:28:53,959 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionGetFramesNeeded
0x2aeeab000    | Fusion               | ERROR | 2023-09-29 05:28:53,959 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionRender
0x30d39f000    | Fusion               | ERROR | 2023-09-29 05:28:53,961 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionGetFramesNeeded
0x30d0bf000    | Fusion               | ERROR | 2023-09-29 05:28:53,962 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionRender
0x30d39f000    | Fusion               | ERROR | 2023-09-29 05:28:53,963 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionGetFramesNeeded
0x2aeeab000    | Fusion               | ERROR | 2023-09-29 05:28:53,964 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionRender
0x30d39f000    | Fusion               | ERROR | 2023-09-29 05:28:53,966 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionGetFramesNeeded
0x2aeeab000    | Fusion               | ERROR | 2023-09-29 05:28:53,966 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionRender
0x2aeeab000    | Fusion               | WARN  | 2023-09-29 05:28:53,966 | Ignoring Fusion dirty notification while in Fusion API call
0x2aeeab000    | Fusion               | ERROR | 2023-09-29 05:28:53,968 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionGetFramesNeeded
0x30d0bf000    | Fusion               | ERROR | 2023-09-29 05:28:53,969 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionRender
0x30d0bf000    | Fusion               | ERROR | 2023-09-29 05:28:53,971 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionGetFramesNeeded
0x30d0bf000    | Fusion               | ERROR | 2023-09-29 05:28:53,971 | Error: nl.smslv.gyroflowofx.fisheyestab_v1 returned <invalid status> from OfxImageEffectActionRender
```

**Comment by AdrianEddy** [comment]
Was it working on previous macOS? Did it broke after the update to Sonoma?

**Comment by jimbodude** [comment]
I recently got this Mac used and it came with Sonoma beta, and I don't have another Mac to work with right now.  I can confirm that I've used the plugin successfully before on MacOS earlier than Sonoma, but I can't confirm that is the critical variable.

**Comment by jimbodude** [comment]
I was able to reproduce the problem in virtualOS with this Sonoma base image https://ipsw.me/download/MacBookPro18,3/23A344#google_vignette

I tried to reproduce it with the most recent 2 Ventura images ( https://ipsw.me/download/MacBookPro18,3/22G120 and https://ipsw.me/download/MacBookPro18,3/22G91 ) but DaVinci crashes whenever I added media, even without the plugin...

Any ideas on how I can help dig deeper?

**Comment by AdrianEddy** [comment]
I'll take a look at this soon, I just updated to Sonoma today, but haven't had a chance to try it yet

**Comment by jimbodude** [comment]
Cool - let me know if I can help with anything.

**Comment by jimbodude** [comment]
#49 suggests that the plugin functions in Sonoma with a 2019 Intel Mac.  I'm on a 2021 M1, so perhaps this is M1-specific, or I missed some configuration.

**Comment by AdrianEddy** [comment]
I just tested on Resolve 18.6 on Sonoma and it works for me, please test latest dev build: https://gyroflow.xyz/devbuild/

**Comment by jimbodude** [comment]
Thanks for checking.  The dev build plugin appears to be working for me...

**Comment by jimbodude** [comment]
Thanks for your help!

**Comment by orr-acle** [comment]
I´ve the same problem: OFX plugin loads, but no effect on video.

I tried Gopro 8:7 and 16:9 video with gopro stabilization disabled - Gyroflow standalone works perfectly fine

I tried "load for current file" as well as import an exported project-data file from standalone app.
- no effect | "stabilization overview" has also no effect

I use davinci resolve studio v18.6
I tried the latest stable/nightly/dev version from the .ofx plugin

Bug or is there anything I missed?
